sdk/digitaltwins/azure-digitaltwins-core/src/main/java/com/azure/digitaltwins/core/implementation/serializer/DigitalTwinsStringSerializer.java

```diff
@@ -33,8 +34,8 @@ public void serialize(String s, JsonGenerator jsonGenerator, SerializerProvider
 
     private boolean isValidJson(String jsonInString ) {
```

Comment:
Is there any way to unit test this? It feels to me it would be more efficient to be able to do that, rather than have to construct models, etc.

Reply:
I like that idea, I will add unit tests for this.